### DISCUSSION TITLE
Is there support for export/import via CSV/excel?

### DISCUSSION THREAD

**Discussion by adamcollingburn** [silevis/reactgrid#276] [Q&A] [answered]
Hey, 

We're currently using ReactGrid in our app and need to start adding import/export functionality

**Comment by Michota** [comment, marked as answer]
And how can we help?

RG (ReactGrid) stores cells data in store, and based on data that each cell, RG will render this data in grid, that contains cells rendered ad separate div's. If you want to create export functionality, I would suggest you to create function that will read and export this data to external file, that can be readen by [Excel API](https://learn.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-ranges-cut-copy-paste). Note, that [Excel API has no "cell" information, but a `Range` object](https://learn.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-cells).

**Comment by qiufeihong2018** [comment]
This ability is very good. It is a very common requirement. Does it support it in v5?
